src/Symfony/Component/Runtime/README.md

```diff
@@ -0,0 +1,114 @@
+Runtime Component
+=================
+
+Symfony Runtime enables decoupling apps from global state.
+
+Getting Started
+---------------
+
+```
+$ composer require symfony/runtime
+```
+
+RuntimeInterface
+----------------
+
+The core of this component is the `RuntimeInterface` which describes a high-order
+runtime logic.
+
+It is designed to be totally generic and able to run any application outside of
+the global state in 6 steps:
+
+ 1. your front-controller returns a closure that wraps your app
+ 2. `RuntimeInterface::resolve()` is given this closure and returns a new closure
+    that wraps the original one but can be called without any arguments (the goal
+    of this step is to resolve the arguments of the original closure, typically by
+    creating domain objects from PHP superglobals;)
```

Comment:
```suggestion
    creating domain objects from PHP superglobals);
```

Reply:
Don't be sad );, mine is happier ;)
More seriously, I don't know which one is correct.

Reply:
https://www.quora.com/Punctuation-In-a-sentence-containing-a-colon-semicolon-placed-alongside-parenthesis-does-the-colon-semicolon-belong-inside-or-outside-the-closed-bracket

Reply:
Thanks for the link! Updated.